### ISSUE TITLE
Homepage content missing

### ISSUE THREAD

**Issue by cameron-doyle** [mirnovov/obsidian-homepage#67]
### Details

When enabling Pin on Open and setting Opening Method to "Replace all open notes" the homepage content will be deleted if the graph is open when closing the window.

Obsidian Desktop
Windows 10

### Steps to reproduce

1. Open homepage.
2. ctrl+G to open graph.
3. Close window.
4. Open obsidian.

### Debug information

```shell
{"version":3,"homepages":{"Main Homepage":{"value":"Dashboard","kind":"File","openOnStartup":true,"openMode":"Replace all open notes","manualOpenMode":"Replace all open notes","view":"Reading view","revertView":false,"openWhenEmpty":true,"refreshDataview":false,"autoCreate":false,"autoScroll":false,"pin":true,"commands":[],"alwaysApply":false}},"separateMobile":false,"_defaultViewMode":"preview","_livePreview":true,"_focusNewTab":"default","_plugins":["obsidian-excalidraw-plugin","obsidian-icon-folder","quickadd","dataview","homepage","cmdr","hide-folders"],"_internalPlugins":["file-explorer","global-search","switcher","graph","backlink","canvas","outgoing-link","tag-pane","properties","page-preview","daily-notes","templates","note-composer","command-palette","editor-status","bookmarks","outline","word-count","file-recovery","sync"],"_obsidianVersion":"1.4.5"}
```

### Checklist

- [X] I updated to the latest version of the plugin.

**Comment by mirnovov** [comment]
I can't reproduce this, but this should theoretically be fixed by 3d609ebfce7fdcdf9f6d2f39c60df860243a0c75 in release 3.5.2. Let me know if it isn't.

**Comment by cameron-doyle** [comment]
The issue has been fixed, Thanks for the hard work.
